mm2src/coins/eth.rs

```diff
@@ -524,6 +529,23 @@ impl EthCoinImpl {
     }
 }
 
+async fn get_raw_transaction_impl(coin: EthCoin, req: RawTransactionRequest) -> RawTransactionResult {
+    let mut tx = req.tx_hash;
+    if tx.len() < 2 {
+        return MmError::err(RawTransactionError::InvalidHashError(tx));
+    }
+    if tx.starts_with("0x") {
+        tx = (tx[2..]).to_string();
```

Comment:
Thanks for @artemii235 note. We can avoid unnecessary cloning and checking the `req.tx_hash` string by using `strip_prefix`:
```rust
let tx = match req.tx_hash.strip_prefix("0x") {
        Some(tx) => tx,
        None => &req.tx_hash,
    };
```